gui/app/main.js

```diff
@@ -292,26 +292,23 @@ const removeFile = (info) => {
   persistLastFiles()
 }
 
-const sendDiskSpace = () => {
+const sendDiskUsage = () => {
   if (diskTimeout) {
     clearTimeout(diskTimeout)
     diskTimeout = null
   }
   if (mainWindow) {
-    diskTimeout = setTimeout(sendDiskSpace, 10 * 60 * 1000)  // every 10 minutes
-    desktop.getDiskSpace((err, res) => {
-      if (err) {
-        console.error(err)
-      } else if (res.diskSpace) {
+    diskTimeout = setTimeout(sendDiskUsage, 10 * 60 * 1000)  // every 10 minutes
+    desktop.diskUsage().then(
+      (res) => {
         const space = {
-          used: +res.diskSpace.usedDiskSpace,
-          usedUnit: res.diskSpace.usedUnit,
-          total: +res.diskSpace.totalDiskSpace,
-          totalUnit: res.diskSpace.totalUnit
+          used: +res.attributes.used,
+          quota: +res.attributes.quota
         }
         sendToMainWindow('disk-space', space)
-      }
-    })
+      },
+      (err) => console.error(err)
```

Comment:
Well, I just transposed the code to replace `getDiskSpace` to `diskUsage` and use promises, without too much thinking. The goal was to have a GUI that doesn't crash